src/overlay_params.h

```diff
@@ -225,6 +228,7 @@ struct overlay_params {
    unsigned media_player_color;
    unsigned table_columns;
    bool no_small_font;
+   bool fcat;
```

Comment:
Unused probably, already in `overlay_params::enabled[OVERLAY_PARAM_ENABLED_fcat]`

Reply:
Yeah, this is due to me misunderstanding how the parameters work. I'll change it.